### PR TITLE
Allows OMX files to be exported to CSV

### LINKED ISSUE #227: Saving log and exporting files to CSV are not working
When I was working on Issue #222, I noticed that some other QAequilibraE tools are not saving/exporting files as they should.

1. **Project -> logfile -> save to disk** - external window to allow the user to select where he/she wants to save the log is not opening, thus saving is not taking place.
2. **Data -> Display project data -> *Select a matrix* -> Load matrix -> Export** - external window to allow the user to select where he/she wants to save the project matrix opens up, but after typing the desired file name, and pressing *save*, the following error pops up:

AequilibraE - Um erro ocorreu enquanto executava o código Python_

### PR DESCRIPTION
**Pull request by r-akemii**
Solves #227. Reactivates the _export_ button that was previously disabled, and now the user can export OMX files to CSV,

### CODE REVIEW

File: qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py [hunk 1]
```diff
@@ -92,8 +92,13 @@ def continue_with_data(self):
         if self.data_type == "AEM":
             self.data_to_show.computational_view([self.data_to_show.names[0]])
         elif self.data_type == "OMX":
+            matrix_index = np.array(list(self.omx.mapping(self.list_indices[0]).keys()))
+            self.data_to_show.create_empty(file_name=os.path.join(tempPath(), "matrix.aem"),
```
Comment (pedrocamargo) [inline qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py]: Having a fixed file in disk for this is a recipe for disaster. AequilibraE has memory-only matrices. Let's use them.

File: qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py [hunk 2]
```diff
@@ -185,8 +190,13 @@ def change_matrix_cores(self):
             self.data_to_show.set_index(idx)
             self.format_showing()
         elif self.data_type == "OMX":
+            matrix_index = np.array(list(self.omx.mapping(idx).keys()))
+            self.data_to_show.create_empty(file_name=os.path.join(tempPath(), "matrix.aem"),
```
Comment (pedrocamargo) [inline qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py]: Repeated code down here and with the same problems

File: qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py (hunk 1 above)
Comment (r-akemii) [reply qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py]: You mean by setting the memory_mode argument?

File: qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from qgis.PyQt.QtWidgets import QComboBox, QCheckBox, QSpinBox, QLabel, QSpacerItem
 from qgis.PyQt.QtWidgets import QHBoxLayout, QTableView, QPushButton, QVBoxLayout
 from qaequilibrae.modules.common_tools import DatabaseModel, NumpyModel, GetOutputFileName
-from qaequilibrae.modules.common_tools.auxiliary_functions import standard_path
+from qaequilibrae.modules.common_tools.auxiliary_functions import standard_path, tempPath
```
Comment (pedrocamargo) [inline qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py]: Are you still using all these imports?

Comment (r-akemii) [reply qaequilibrae/modules/matrix_procedures/display_aequilibrae_formats_dialog.py]: Only _standard_path_ was in the code before. _tempPath_ was an addition that I forgot to remove.

**Review by pedrocamargo — APPROVED** [review]

**Pull request merged**